diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index b5ba130e6ed00e54d76884ff305d4d6566337781..a434177edc99d096f928fb03f94ef88b0c1b201f 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -364,6 +364,7 @@ void ToolbarView::Init() {
   } else {
     // TODO(crbug.com/932818): Remove this once the
     // |kAutofillEnableToolbarStatusChip| is fully launched.
+    BRAVE_TOOLBAR_VIEW_INIT
     avatar_ =
         AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
     avatar_->SetVisible(show_avatar_toolbar_button);
